Formula/vice.rb

```diff
@@ -25,52 +28,88 @@ class Vice < Formula
   depends_on "libogg"
   depends_on "libpng"
   depends_on "libvorbis"
+  depends_on "mpg123"
   depends_on "portaudio"
-  depends_on "sdl2"
   depends_on "xz"
+  depends_on "sdl2"
 
-  # needed to avoid Makefile errors with the vendored ffmpeg 2.4.2
-  resource "make" do
-    url "https://ftp.gnu.org/gnu/make/make-4.2.1.tar.bz2"
-    mirror "https://ftpmirror.gnu.org/make/make-4.2.1.tar.bz2"
-    sha256 "d6e262bf3601b42d2b1e4ef8310029e1dcf20083c5446b4b7aa67081fdffc589"
-  end
+  patch :DATA
```

Comment:
Can we include a link to where this patch has been submitted upstream? Thanks!

Reply:
I think he meant to include it as a comment in the Formula so that it can be investigated at a later date if it can be removed.